app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class QuotesController < ApplicationController
+  def index
+    quotes_arr = QuotesFacade.create_quotes(params[:query])
+    @quotes = quotes_arr[0]
+    @total_count = quotes_arr[1]
+  end
```

Comment:
sweet! I like that you're getting both of these from the same method since they come back from the same api call. I know in your facade you asked for feedback on this. And what you did was a completely acceptable way to send back two things at once. I wouldn't say your facade is unconventional, but I'd say you could update this action to accept and define the two values you return from your facade .

So, if you kept your facade method EXACTLY as it is right now, and set your return value in this action to two variables, ruby will automatically assign the array to the respective variable: 

```ruby
@quotes, @total_count = QuotesFacade.create_quotes(params[:query])
```